DOCS.md

```diff
@@ -0,0 +1,18 @@
+Use the GitHub Release plugin for publishing files and artifacts to GitHub
```

Comment:
Maybe just plugin for publishing and artifcats to GitHub? releases sounds redundant.